OCPBUGS-45841:clarifies overlapping ANP rules (#85972)
openshift/openshift-docs: modules/nw-ovn-k-anp-best-practices.adoc

When building `AdminNetworkPolicy` (ANP) resources, you might consider the following when creating your policies:

> * You can create ANPs that have the same priority. When creating two ANPs at the same priority make sure they do not apply overlapping rules to the same traffic because there is no guarantee which rule will get applied. Because there is no guarantee which policy will take precedence when overlapping ANPs are created, it is preferable to create ANPs at different priorities so that precedence is well defined.

Reviewer: ```suggestion
* You can create ANPs that have the same priority. If you do create two ANPs at the same priority, ensure that they do not apply overlapping rules to the same traffic. Only one rule per value is applied and there is no guarantee which rule is applied when there is more than one at the same priority value. Because there is no guarantee which policy takes precedence when overlapping ANPs are created, set ANPs at different priorities so that precedence is well defined.
```
OK, so ANPs only apply one per value, no matter what? 
"set" or "configure" maybe to reduce "create" redundancy?

Reviewer 2: @JoeAldinger It is not ANPs that are overlapping but the rules defined in the ANP are the ones that are overlapping.

Author: @asood-rh the rules are part of the ANP so I _think_ we can say that the ANP are overlapping if the part of the whole is overlapping. I'll re-write for clarity based on Shauna's feedback.
